src/__tests__/utils/arithmetic-helper.test.ts

```diff
@@ -0,0 +1,490 @@
+    //   const input = { param1: '=10/0' };
+  
+    //   expect(() => {
+    //     const response = evaluateInput(input);
+    //     console.log(response)
+    //   }).toThrow(
+    //     new Error('Division by zero is not allowed in arithmetic expressions.')
+    //   );
+    // });
+  })
+
+  describe('evaluateConfig():', () => {
+      // it('should evaluate a simple arithmetic expression in config', () => {
+      //   const options = {
+      //     config: { param1: '=2*5' },
+      //     input: {},
+      //     parametersToEvaluate: ['param1']
+      //   };
+    
+      //   const result = evaluateConfig(options);
+    
+      //   expect(result).toEqual({ param1: 10 });
+      // });
+    
+      it('should skip evaluation if parameter is not in parametersToEvaluate', () => {
+        const options = {
+          config: { param1: '=2*5', param2: '10' },
+          input: {},
+          parametersToEvaluate: []
+        };
+    
+        const result = evaluateConfig(options);
+    
+        expect(result).toEqual({ "param1": "=2*5", param2: '10' });
+      });
+    
+      it('should throw error for invalid arithmetic expression', () => {
+        const options = {
+          config: { param1: '=2^5' },
+          input: {},
+          parametersToEvaluate: ['param1']
+        };
+    
+        expect(() => evaluateConfig(options)).toThrow(
+          "The `param1` contains an invalid arithmetic expression. It should start with `=` and include the symbols `*`, `+`, `-` and `/`."
+        );
+      });
+    
+      it('should use values from input in arithmetic expression', () => {
+        const options = {
+          config: { param1: '=inputValue*2' },
+          input: { inputValue: 5 },
+          parametersToEvaluate: ['param1']
+        };
+    
+        const result = evaluateConfig(options);
+    
+        expect(result).toEqual({ param1: 10 });
+      });
+    
+      it('should return original config if no parameters need evaluation.', () => {
+        const options = {
+          config: { param1: '10', param2: '20' },
+          input: {},
+          parametersToEvaluate: []
+        };
+    
+        const result = evaluateConfig(options);
+    
+        expect(result).toEqual({ param1: '10', param2: '20' });
+      });
+    });
+
+    describe('isValidArithmeticExpression(): ', () => {
+      it('should return true for a valid arithmetic expression with variables.', () => {
+        const parameter = '=param1*2';
+        const result = isValidArithmeticExpression(parameter);
+
+        expect(result).toBe(true);
+      });
+    
+      it('should return true for a valid arithmetic expression with variables and numbers.', () => {
+        const parameter = '=param1+5/2';
+        const result = isValidArithmeticExpression(parameter);
+
+        expect(result).toBe(true);
+      });
+    
+      it('should return false for an expression without the "=" sign.', () => {
```

Comment:
this case is valid, it should return true

```suggestion
      it('should return true for an expression without the "=" sign.', () => {
```